src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -146,12 +148,7 @@ public void initialize() {
   }
 
   public void rezeroIntake() {
-    deployMotorEncoder.setPosition(
-        AngleUtil.wrapAngle(
-            deployMotorAbsoluteEncoder.getPosition()
-                - Cal.Intake.ABSOLUTE_ENCODER_START_POS_DEG
-                + Cal.Intake.STARTING_POSITION_DEGREES));
-
+    deployMotorEncoder.setPosition(deployMotorAbsoluteEncoderChecker.getMedian());
```

Comment:
So `deployMotorAbsoluteEncoderChecker.getMedian()` really only replaces `deployMotorAbsoluteEncoder.getPosition()`, we still need the other parts